stage #2.5/module #3 Spring Core/#5.2. Java-based configuration/content.md

```diff
@@ -0,0 +1,320 @@
+<h1 style="color: green">Java-based Container Configuration</h1>
+
+The central artifacts in Spring's Java-configuration support are `@Configuration`-annotated classes and
+`@Bean`-annotated methods.
```

Comment:
Move content from `#5.2. Classpath Scanning and Managed Components` 129-162 lines here

Reply:
Could you please clarify the reason of doing so, thanks.

Reply:
I thought that it would be a good example of java-based annotation, but since you in "Classpath Scanning and Managed Components" describes all approaches to create a bean, it's better to leave it in initial place